qunicorn_core/db/cli.py

```diff
@@ -85,12 +86,19 @@ def get_quasm_string() -> str:
 
 def load_db_function(app: Flask):
     user = UserDataclass(name="DefaultUser")
-    qc = QuantumProgramDataclass(quantum_circuit=utils.get_default_qasm_string(1))
-    qc2 = QuantumProgramDataclass(quantum_circuit=utils.get_default_qasm_string(2))
+    qc = QuantumProgramDataclass(
+        quantum_circuit=utils.get_default_qasm_string(1), assembler_language=AssemblerLanguage.QASM
+    )
+    qc2 = QuantumProgramDataclass(
+        quantum_circuit=utils.get_default_qasm_string(2), assembler_language=AssemblerLanguage.QASM
+    )
     qasm3_str: str = "OPENQASM 3; \nqubit[3] q;\nbit[3] c;\nh q[0];\ncnot q[0], q[1];\ncnot q[1], q[2];\nc = measure q;"
-    qasm3_program = QuantumProgramDataclass(quantum_circuit=qasm3_str)
+    qasm3_program = QuantumProgramDataclass(quantum_circuit=qasm3_str, assembler_language=AssemblerLanguage.QASM)
     braket_str: str = "Circuit().h(0).cnot(0, 1)"
-    braket_program = QuantumProgramDataclass(quantum_circuit=braket_str)
+    braket_program = QuantumProgramDataclass(quantum_circuit=braket_str, assembler_language=AssemblerLanguage.BRAKET)
+
+    qiskit_str: str = "global qc; qc = QuantumCircuit(2, 2);qc.h(0);qc.cx(0, 1);qc.measure(0, 0);qc.measure(1, 1)"
```

Comment:
check if possible easier option might be possible

Reply:
TODO is added to the ticket